### PR TITLE
add bert zero

### PR DESCRIPTION
**Pull request by Wesley-Jzy**

### CODE REVIEW

File: bert/colossalai_utils/utils.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    from colossalai.core import global_context as gpc
+    from colossalai.logging import disable_existing_loggers
+    from colossalai.nn.optimizer import CPUAdam
+    from colossalai.zero.init_ctx import ZeroInitContext
+    from colossalai.zero.shard_utils import (
+                                             TensorShardStrategy)
+    from colossalai.zero.sharded_model import ShardedModel, ShardedModelV2
+    from colossalai.zero.sharded_optim import ShardedOptimizerV2
+
+    disable_existing_loggers()
```
Comment (feifeibear) [inline bert/colossalai_utils/utils.py]: ？enable logger?

File: bert/colossalai_utils/bert_config_zerov1.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        "growth_interval": 1000
+    },
+    "gradient_clipping": 0.0,
+    "zero": {
+        "mixed_precision": true, 
+        "reshard_after_forward": false, 
+        "offload_config" : {
+            "device": "cpu"
+        },
+        "version": 1
```
Comment (feifeibear) [inline bert/colossalai_utils/bert_config_zerov1.json:24]: use version 2 please.
version 1 is aborted

File: zero/run.py [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-from common.gpt2 import gpt2_builder
-from common.train import train
-from common.utils import CONFIG, load_config, print_log
-from common.vit import vit_builder
-from deepspeed_utils.utils import init_w_ds
-from fairscale_utils.utils import init_w_fs
-from patrickstar_utils.utils import init_w_ps
-from torch_utils.utils import init_w_torch
+from zero.colossalai_utils.utils import init_w_col
+from zero.common.gpt2 import gpt2_builder
```
Comment (feifeibear) [inline zero/run.py:4]: did the builder really builds a bert model?

File: bert/colossalai_utils/bert_config_zerov1.json (hunk 2 above)
Comment (Wesley-Jzy) [reply bert/colossalai_utils/bert_config_zerov1.json:24]: Done

File: bert/colossalai_utils/bert_config_zerov1.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    "use_mem_monitor": true,
+    "sharded_model_version": 1
```
Comment (feifeibear) [inline bert/colossalai_utils/bert_config_zerov1.json:27]: remove this line

**Review by feifeibear — APPROVED** [review]

**Review by feifeibear — APPROVED** [review]

File: zero/run.py (hunk 3 above)
Comment (Wesley-Jzy) [reply zero/run.py:4]: It's gpt zero's run.py. Just change the import path for it.

File: bert/colossalai_utils/bert_config_zerov1.json (hunk 4 above)
Comment (Wesley-Jzy) [reply bert/colossalai_utils/bert_config_zerov1.json:27]: Necessary to my v1 config, I removed it from v2 config.
Done.

File: bert/colossalai_utils/utils.py (hunk 1 above)
Comment (Wesley-Jzy) [reply bert/colossalai_utils/utils.py]: Done.

**Pull request merged**